desktop/components/credit_card/stylesheets/index.styl

```diff
@@ -50,6 +50,7 @@
         margin-right: 6%
       .error
         position absolute
+        top 4px
```

Comment:
Adding this to address some funky error placement:
![image](https://cloud.githubusercontent.com/assets/9088720/24678667/729d4162-1959-11e7-8897-0a5960eb8f83.png)
![image](https://cloud.githubusercontent.com/assets/9088720/24678675/7c905132-1959-11e7-9b71-f13f48655e1d.png)